src/app/components/Transactions/index.tsx

```diff
@@ -75,23 +76,27 @@ export const Transactions: FC<TransactionProps> = ({
       },
       {
         content: (
-          <TrimLinkLabel
-            label={transaction.hash}
-            to={RouteUtils.getTransactionRoute(transaction.hash, ParaTime.Emerald)}
-          />
+          <Typography variant="mono">
+            <TrimLinkLabel
```

Comment:
Wouldn't it make sense to use the Typography inside the link component?

Reply:
I've done it this way initially (inside the TrimLabel), but moved typography outside the component. Wasn't sure if it's really belong there. Anyway, will gather additional feedback before merging.

Reply:
From the usage perspective I prefer if the `Typography` is outside, and the only thing that is actually clickable is the content itself. `Typography` could have some padding, which would make "empty" space clickable.